R/kallisto.R

```diff
@@ -29,12 +29,11 @@ kallisto_counts <- function(x, tx2gene = NULL) {
   txi_kallisto <- tximport::tximport(files = x, type = "kallisto", tx2gene = tx2gene)
   counts <- txi_kallisto[["counts"]] |>
     tibble::as_tibble(rownames = "rowname", .name_repair = make.names) |>
-    dplyr::rename(count = X)
+    dplyr::rename(count = X) |>
+    dplyr::filter(!grepl("PAR_Y", .data$rowname))
 
   counts <- counts |>
     dplyr::mutate(rowname = sub("\\..*", "", .data$rowname)) |>
-    dplyr::mutate(across(
-      .cols = matches('count'),
-      .fns = ~ as.integer(.x)))
+    dplyr::mutate(count = as.integer(.data$count))
```

Comment:
You can combine multiple mutates in one instead of duplicating e.g. https://dplyr.tidyverse.org/reference/mutate.html#ref-examples

I tend to use separate calls only when the expression is complex.